bot/seasons/evergreen/bookmark.py

```diff
@@ -19,15 +21,18 @@ def __init__(self, bot: commands.Bot):
     async def bookmark(
         self,
         ctx: commands.Context,
-        target_message: discord.Message,
+        target_message: discord.Message = None,
         *,
         title: str = "Bookmark"
     ) -> None:
-        """Send the author a link to `target_message` via DMs."""
+        """Send the author a link to `target_message` via DMs. If Nothing is provided it take the commanding message."""
+        if target_message is None:
+            target_message = ctx.message
```

Comment:
By doing this, now when someone do `.bm` instead of getting a help message, the bot will bookmark the message with `.bm` instead like here

![image](https://user-images.githubusercontent.com/19921765/77986719-06f55a80-7342-11ea-8246-134e7dd4d45e.png)

![image](https://user-images.githubusercontent.com/19921765/77986728-0c52a500-7342-11ea-9850-82c75b2effe2.png)

There need to be a stricter check here to prevent this to happen.

Reply:
I mean anyone can use .help bm to get the usage. 
For now it looks clean and easy to remember. One cannot remember a special word for just a command.

Or maybe you can suggest some better checks ?

Reply:
The problem is not that we have a `.help bm` - the problem is that `.bm` by itself, without any arguments, will work.

So going by that, you can definitely check the part where there is no message being passed to it - when it is `None`

Reply:
That is the scope of #338 and that is why it is there.

Reply:
Right now it is not doing what is said in #338 at all, like in the screenshot, it is bookmarking the **current message invoking the command** which is the message with `.bm`

Reply:
Latest commit make it better . And .bm
Now invoke its usage error